web/src/ui_util.ts

```diff
@@ -343,3 +343,14 @@ export function get_left_sidebar_search_term(): string {
     assert(search_term !== undefined);
     return search_term.trim();
 }
+
+export function disable_left_sidebar_search(): void {
+    if ($<HTMLInputElement>("#left-sidebar-search input").val()) {
+        $("#left-sidebar-search .input-close-filter-button").trigger("click");
```

Comment:
Maybe add a comment clarifying this is triggering a global input click handler, which is why we're not just calling a close function directly.

Reply:
Done.